javascript/src/input_slider.js

```diff
@@ -90,26 +90,20 @@ input.register({
         ctHandle.set(keys);
         lastKnownKeys = keys;
       }
-    });
-
-
-    // let $el = $(el);
-    // $el.on("change", "input[type="checkbox"]", function() {
-    //   let checked = $el.find("input[type="checkbox"]:checked");
-    //   if (checked.length === 0) {
-    //     ctHandle.clear();
-    //   } else {
-    //     let keys = {};
-    //     checked.each(function() {
-    //       data.map[this.value].forEach(function(key) {
-    //         keys[key] = true;
-    //       });
-    //     });
-    //     let keyArray = Object.keys(keys);
-    //     keyArray.sort();
-    //     ctHandle.set(keyArray);
-    //   }
-    // });
+    }
+    $el.on("change.crosstalkSliderInput", updateFilter);
+
+    // Update filter now in case this code happens to execute
+    // after widget(s) are done rendering
+    updateFilter();
+
+    // Schedule another update when all widgets are done rendering
+    // This is especially relevant for `runtime: shiny` where widgets
+    // likely haven't rendered at this point and may only register
+    // FilterHandle.on("change", ...) callbacks in their renderValue
+    if (window.HTMLWidgets) {
+      window.HTMLWidgets.addPostRenderHandler(updateFilter);
+    }
```

Comment:
This'll only work for statically rendered widgets -- we should do https://github.com/rstudio/crosstalk/pull/100 instead

```suggestion
```